### PR TITLE
Moar tests

### PR DESCRIPTION
**Pull request by louislai**
#97 Lol every time I write tests, I detect some bugs

### CODE REVIEW

File: src/main/java/seedu/toluist/controller/AliasController.java [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
  */
 public class AliasController extends Controller {
+    public static final String ALIAS_TERM = "alias";
+    public static final String COMMAND_TERM = "command";
+    public static final String COMMAND_WORD = "alias";
+
     private static final String RESULT_MESSAGE_SUCCESS = "Alias %s for %s was added";
     private static final String RESULT_MESSAGE_FAILURE = "Alias %s for %s could not be added";
-    private static final String RESULT_MESSAGE_RESERVED_WORD = "%s is a reserved word";
+    public static final String RESULT_MESSAGE_RESERVED_WORD = "%s is a reserved word";
```
Comment (Melvin-Tan) [inline src/main/java/seedu/toluist/controller/AliasController.java:22]: Why out of all the strings only this is public?

File: src/main/java/seedu/toluist/controller/ExitController.java [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 
 /**
  * Handle exit command
  */
 public class ExitController extends Controller {
-    private static final String COMMAND_TEMPLATE = "^(exit|quit)$";
-    private static final String COMMAND_WORD_EXIT = "exit";
-    private static final String COMMAND_WORD_QUIT = "quit";
+    public static final String COMMAND_WORD_EXIT = "exit";
+    public static final String COMMAND_WORD_QUIT = "quit";
```
Comment (Melvin-Tan) [inline src/main/java/seedu/toluist/controller/ExitController.java:13]: Are these two 'public' intentional? If yes I have no issues here

File: src/main/java/seedu/toluist/controller/UnaliasController.java [hunk 3]
```diff
@@ -15,7 +15,7 @@
 public class UnaliasController extends Controller {
     private static final String RESULT_MESSAGE_SUCCESS = "Alias %s has been removed";
     private static final String RESULT_MESSAGE_FAILURE = "Alias %s cannot be removed";
-    private static final String RESULT_MESSAGE_NOT_ALIAS = "%s is not an alias";
+    public static final String RESULT_MESSAGE_NOT_ALIAS = "%s is not an alias";
```
Comment (Melvin-Tan) [inline src/main/java/seedu/toluist/controller/UnaliasController.java:18]: This too. public? Or should make it private like RESULT_MESSAGE_SUCCESS then handle things using functions

File: src/main/java/seedu/toluist/model/Task.java [hunk 4]
```diff
@@ -90,8 +92,8 @@ public void setCompleted(boolean isCompleted) {
     }
 
     public void setDeadLine(LocalDateTime deadLine) {
-        setEndDateTime(null);
-        setStartDateTime(deadLine);
+        setEndDateTime(deadLine);
+        setStartDateTime(null);
```
Comment (Melvin-Tan) [inline src/main/java/seedu/toluist/model/Task.java]: Not that this matters but maybe should set start date then end date (i.e. flip the two lines)

File: src/test/java/seedu/toluist/model/TaskTest.java [hunk 5]
```diff
@@ -24,108 +31,194 @@
...
+        // task is now an event
+        assertTrue(testTask.isEvent());
+    }
+
+    @Test
+    public void setComplete() {
+        LocalDateTime now = LocalDateTime.now();
+        // Set completed
+        testTask.setCompleted(true);
+        LocalDateTime a = testTask.getCompletionDateTime();
```
Comment (Melvin-Tan) [inline src/test/java/seedu/toluist/model/TaskTest.java]: remove, or test a.

File: src/main/java/seedu/toluist/controller/ExitController.java (hunk 2 above)
Comment (louislai) [reply src/main/java/seedu/toluist/controller/ExitController.java:13]: yes. To facilitate testing

**Review by Melvin-Tan — APPROVED** [review]

**Pull request merged**